tool_collections/samtools/bam_to_cram/samtools_bam_to_cram.xml

```diff
@@ -0,0 +1,125 @@
+<tool id="samtools_bam_to_cram" name="samtools BAM to CRAM" version="1.2.0">
+    <description>convert BAM alignments to CRAM format</description>
+
+    <macros>
+        <import>macros.xml</import>
+    </macros>
+    <expand macro="requirements"></expand>
+    <expand macro="version_command"></expand>
+    <expand macro="stdio"></expand>
+
+    <command>
+    <![CDATA[
+        #if str( $input_alignment.metadata.bam_index ) != "None":
+            ln
+                -f
+                -s
+                "${input_alignment.metadata.bam_index}"
+                "${input_alignment}.bai"
+                &&
+        #end if
+
+        samtools view
+            #if $parameter_regions.target_region == "regions_bed_file"
+                -L $parameter_regions.regions_bed_file
+            #end if
+            
+            -@\${GALAXY_SLOTS:-2}
+            -C
+            -T ${reference_source.input_reference}
+            -o $output_alignment
+            $input_alignment
```

Comment:
`${reference_source.input_reference}`, `$output_alignment` and `$input_alignment` should be quoted.